packages/PriceLockup/PriceLockup.md

```diff
@@ -28,3 +40,55 @@
 ```jsx
 <PriceLockup size="medium" topText="À partir de :" price="25" signDirection="right" />
 ```
+
+### <a name="footnoteLinkWithPriceLockup">PriceLockup with FootnoteLink</a>
+
+Usage criteria:
+
+- Display `FootnoteLink` at the end of `rateText` or `bottomText`
+- Group multiple `FootnoteLinks` together so that they:
+  - Display in the same line as the `rateText` or `bottomText` when there’s available space,
+  - Or display below `rateText` or `bottomText`, whichever is the last element
```

Comment:
It feels like this should grammatically be part of the last point. (Since the last point ends with a comma and this one starts with "Or") Maybe they can be re-written to be more separate?